app/models/spotlight/exhibit.rb

```diff
@@ -108,5 +109,9 @@ def reindex_progress
     def sanitize_description
       self.description = ::Rails::Html::FullSanitizer.new.sanitize(description)
     end
+
+    def new_reindexing_log_entry(user = nil)
+      Spotlight::ReindexingLogEntry.create(exhibit: self, user: user, items_reindexed_count: resources.size, job_status: 'unstarted')
```

Comment:
Yeah, `resources.size` is the count of the explicitly provided resources, not the count including items in the collection. Note too that you won't want to calculate it synchronously either, since it can be expensive to calculate.